inc/toolbox.class.php

```diff
@@ -2828,7 +2828,8 @@ public static function formatOutputWebLink($link) {
     */
    public static function useCache() {
       global $GLPI_CACHE;
-      return $GLPI_CACHE !== false && (!defined('TU_USER') || defined('CACHED_TESTS'));
+      return !($GLPI_CACHE instanceof Zend\Cache\Storage\Adapter\AbastractAdapter)
```

Comment:
bad negation ?

Reply:
I don't think so (removing negation make tests to fail), but I'll check if the second part is OK :)

Reply:
I had a typo in the class name... ("Abastract")